README.md

```diff
@@ -485,26 +484,6 @@ Alternatively, if your playbook **only runs on Windows hosts**, use the followin
 
 ## Troubleshooting
 
-### Debian stretch
```

Comment:
Why was this removed? If it's no longer needed, maybe we should mention the versions in which this happened and that upgrading should fix it? People might not look into the docs of their playbook version

Reply:
Yeah, you're right about leaving it there and mentioning versions to which this applied.